pkg/virtual/workspaces/registry/rest_test.go

```diff
@@ -1139,14 +1140,6 @@ func TestDeleteWorkspace(t *testing.T) {
 			assert.NoError(t, err)
 			assert.Nil(t, response)
 			assert.False(t, deletedNow)
-			crbList, err := kubeClient.Tracker().List(rbacv1.SchemeGroupVersion.WithResource("clusterrolebindings"), rbacv1.SchemeGroupVersion.WithKind("ClusterRoleBinding"), "")
```

Comment:
Shouldn't we here test that the bindings are still there, instead of simply removing the tests ?

Reply:
@davidfestal good point, will update 👍🏻